docs/en/compute/Google-Dataproc.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: global
+title: Running Alluxio on Google Cloud Dataproc
+nickname: Google Dataproc
+group: Data Applications
+priority: 0
+---
+
+This guide describes how to configure Alluxio to run on [Google Cloud Dataproc](https://cloud.google.com/dataproc).
+
+* Table of Contents
+{:toc}
+
+## Overview
+
+Google Cloud Dataproc is a managed on-demand service to run Spark and Hadoop compute workloads.
+It manages the deployment of various Hadoop Services and allows for hooks into these services for
+customizations.
+Aside from the added performance benefits of caching, Alluxio also enables users to run compute 
+workloads against on-premise storage or even a different cloud provider's storage i.e. AWS, Azure
+Blob Store.
+
+## Prerequisites
+
+* Account with Cloud Dataproc API enabled
+* A GCS Bucket
```

Comment:
I think gcloud has some version of IAM so we don't need to specify credentials. We should mention that that service must be enabled for the project/dataproc instances.

Reply:
Our legacy gs connector does not read credentials from the credential store

Reply:
I see. Then we should mention that interoperability keys are a prerequisite for Alluxio on GCS until we decide if the new GCS UFS will be in OS or not.

Reply:
sounds good